modules/claims_api/app/controllers/claims_api/v2/veterans/intent_to_file_controller.rb

```diff
@@ -8,12 +8,18 @@ module Veterans
       class IntentToFileController < ClaimsApi::V2::ApplicationController
         before_action :verify_access!
 
-        ITF_TYPES = %w[compensation pension burial].freeze
-
+        # GET to fetch active intent to file by type
+        #
+        # @return [JSON] ITF record
         def type
+          itf_types = ClaimsApi::V2::ParamsValidation::IntentToFile::IntentToFileValidator.itf_types
+          unless itf_types.include?(request.path_parameters[:type])
```

Comment:
The validator `.downcase`s this value, should we do that here too? What happens if someone makes a request with `"PENSION"` (all caps)?

Reply:
Current code will treat mixed-case or all caps type parameters as invalid and return a 404 with detail message indicating an invalid type parameter. I can change that if we want to allow otherwise valid type parameters with case differences.

Reply:
Updated to accept valid type parameters with case differences, to match existing validation.